core/docker-entrypoint.sh

```diff
@@ -58,6 +58,13 @@ function setup_database () {
 		exit 1
 	fi
 
+	# Translate mysql:// urls to mysql+mysql:// backend:
+	if [[ "$DATABASE_URL" == mysql://* ]]; then
+		DATABASE_URL="mysql+pymysql://${DATABASE_URL:8}"
+		echo "Database URL was automatically rewritten to:"
+		echo "$DATABASE_URL"
```

Comment:
I think it can be changed to a single line:

```bash
echo "Database URL was automatically rewritten to: $DATABASE_URL"
```

Reply:
I used two lines to not go over an 80 column line length, but I suppose that is probably a silly reason

Reply:
I *mostly* try to follow that rule, but doesn't always makes it very pretty in bash.